oci/oci.go

```diff
@@ -20,6 +20,15 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+const (
+	// ContainerStateCreated represents the created state of a container
+	ContainerStateCreated = "created"
```

Comment:
We could also use uints and a bitmask here to reduce strings comparison, wdyt? Or is this too much future proof lol